crates/solver/src/settlement_access_list.rs

```diff
@@ -66,6 +66,8 @@ pub async fn estimate_settlement_access_list(
     settlement: &Settlement,
     tx: &TransactionBuilder<DynTransport>,
 ) -> Result<AccessList> {
+    tracing::debug!(?settlement, "generating access list for settlement");
```

Comment:
If we make these `trace` logs, we can temporarily enable them by configuration and then disable them once more once they've served their purpose.

Reply:
I'm not sure how Tenderly links support access lists. Just something to keep in mind.